src/App.js

```diff
@@ -38,6 +42,10 @@ function App() {
         <Route path="/seller-portal" element={<SellerPortal />} />
         <Route path="/buyer-portal" element={<BuyerPortal />} />
         <Route path="/properties" element={<PropertyList />} />
+        <Route path="/" element={<HomePage />}/>
+        <Route path="/signuplogin" element={<SignupLogin />}/>
+        <Route path="/logout" element={<Logout />}/>
+        <Route path="/profile" element={<Profile />}/>
```

Comment:
remove this line if you're giving up on all Profile related features in this PR.